.run/Test Profile_ auth-jwt-custody.run.xml

```diff
@@ -1,12 +1,11 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Test Profile: auth-jwt-custody" type="JetRunConfigurationType" folderName="Run Test Profile">
     <envs>
-      <env name="app.enableTest" value="true" />
       <env name="TEST_PROFILE_NAME" value="auth-jwt-custody" />
-      <env name="KT_REFERENCE_SERVER_CONFIG" value="service-runner/src/main/resources/config/reference-config.yaml" />
     </envs>
     <option name="MAIN_CLASS_NAME" value="org.stellar.anchor.platform.run_profiles.RunTestProfile" />
     <module name="java-stellar-anchor-sdk.service-runner.main" />
+    <option name="REDIRECT_INPUT" value="true" />
```

Comment:
Why is this required for this profile?

Reply:
Not sure. This was added by IntelliJ. 

I have removed it manually.